hkmc2/shared/src/test/mlscript-compile/String.mls

```diff
@@ -0,0 +1,6 @@
+module String with ...
+
+fun (++) concat(lhs, rhs) =
```

Comment:
BTW I'm thinking to reserve `++` for proper collections, which is why `Str.mls` uses `~` (to avoid name clashes later on).

Reply:
OK. I'll update.